contrib/ci-workers/README.md

```diff
@@ -3,6 +3,12 @@
 This folder contains the Ansible playbooks for configuring a fresh OS
 installation for use as a Buildbot worker in Zcash's CI.
 
+# Criteria for Adding Workers
+
+a. Don't add workers until users complain about a problem on a platform that doesn't yet have workers or if we anticipate many users will use a platform, we may pre-emptively add unsupported for it.
+b. Prioritize the workers that seem to have the most users. In practice, this might be the ones with the loudest users, since it may be hard to get effective user counts.
```

Comment:
s/workers/platforms/

Drop the second sentence; it sounds critical of the louder users.